app/src/index.js

```diff
@@ -0,0 +1,9 @@
+import './style.css';
```

Comment:
1. You have only added scss loader, so css files won't load correctly
2. For scss files I'm having errors in console. Seems like you've added redundant css loader